.kitchen.yml

```diff
@@ -0,0 +1,96 @@
+---
+driver:
+  name: docker
+  hostname: keystone.ci.local
+  use_sudo: true
+
+provisioner:
+  name: salt_solo
+  salt_install: bootstrap
+  salt_bootstrap_url: https://bootstrap.saltstack.com
+  salt_version: latest
+  require_chef: false
+  log_level: info
+  formula: keystone
+  grains:
+    noservices: False
+  dependencies:
+    - name: linux
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-linux
+    - name: mysql
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-mysql
+    - name: apache
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-apache
+    - name: memcached
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-memcached
+    - name: cadf
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-cadf
+    - name: rabbitmq
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-rabbitmq
+  state_top:
+    base:
+      "*":
+        - linux
+        - mysql
+        - keystone
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - linux_repo_openstack
+          - keystone
+          - mysql_platform_specific
+    mysql_platform_specific.sls: {}
+  pillars-from-files:
+    linux_repo_openstack.sls: tests/pillar/repo_mos9.sls
```

Comment:
Just a heads up: it would be cool to be able to test various openstack distributions (Ubuntu Cloud, MOS, Debian, etc.)